<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Repleisiukas</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QAdvancedTabBar" name="tabs" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="trashButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Å </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QAdvancedInputPanel" name="stringIn" native="true"/>
       <widget class="QTextEdit" name="query">
        <property name="tabStopWidth">
         <number>40</number>
        </property>
       </widget>
      </widget>
      <widget class="QPlainTextEdit" name="stringOut">
       <property name="tabStopWidth">
        <number>40</number>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_Go">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>2</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Paste/Copy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQueris">
    <property name="title">
     <string>Queris</string>
    </property>
    <widget class="QMenu" name="menuLastUsed">
     <property name="title">
      <string>LastUsed</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionLoad"/>
    <addaction name="menuLastUsed"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionAuto_update"/>
    <addaction name="actionDebugger"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
   </widget>
   <addaction name="menuQueris"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionAuto_update">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto update</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="text">
    <string>execute</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Enter</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionDebugger">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debugger</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAdvancedInputPanel</class>
   <extends>QWidget</extends>
   <header>qadvancedinputpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QAdvancedTabBar</class>
   <extends>QWidget</extends>
   <header>qadvancedtabbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
